src/sql/workbench/common/editorReplacerContribution.ts

```diff
@@ -55,6 +56,11 @@ export class EditorReplacementContribution implements IWorkbenchContribution {
 			language = this.modeService.getModeIdByFilepathOrFirstLine(editor.getResource());
 		}
 
+		if (!language) {
```

Comment:
can you replace an editor with another untitled editor?

Reply:
What do you mean?